README.md

```diff
@@ -101,6 +101,18 @@ repositories {
 
 MMKV-KTX greatly simplifies the usage of key-value storage in Kotlin. As a result, you can almost immediately understand how to use it just by looking at the following example.
 
+#### Initialization
+
+Before using it, you need to initialize the MMKV instance.
```

Comment:
Before starting, please make sure you have initialized the [MMKV](https://github.com/Tencent/MMKV#quick-tutorial) instance. If not, initialize it anywhere that can access the `Context`, such as in the `Application`: